Auto Birthday Wisher/Auto B'Day Wisher.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd                                                     # Pandas library is used for importing and reading the data
+import datetime                                                         # datetime module is used for fetching the dates
+import smtplib															# smtp library used for sending mail
+import os
+
+current_path = os.getcwd()
+print(current_path)
+os.chdir(current_path)                                                  # Changing the Path of the directory in which you are currently working
+
+GMAIL_ID = '</ Enter your email here />'                                # Give your mail here from which you want to send the wishes
+GMAIL_PSWD = '</ Enter password for your email here />'                  # Give your mail password
+
+
+def sendEmail(to, sub, msg):
+    print(f"Email to {to} sent: \nSubject: {sub} ,\nMessage: {msg}")
+    s = smtplib.SMTP('smtp.gmail.com', 587)                             # creating server to send mail
+    s.starttls()                                                        # start a TLS session
+    s.login(GMAIL_ID, GMAIL_PSWD)                                       # the function will login with your Gmail credentials
+    s.sendmail(GMAIL_ID, to, f"Subject: {sub} \n\n {msg}")              # sending the mail
+    s.quit()
+
+
+if __name__ == "__main__":
+    df = pd.read_excel("data.xlsx")                                     # the datasheet where the data of the friends is stored
+    print(df)
+    today = datetime.datetime.now().strftime("%d-%m")
+    yearNow = datetime.datetime.now().strftime("%Y")
+
+    writeInd = []
+    for index, item in df.iterrows():
+        bday = item['Birthday']
+        bday = datetime.datetime.strptime(bday, "%d-%m-%Y")
+        bday = bday.strftime("%d-%m")
+        print(bday)
+        if(today == bday) and yearNow not in str(item['LastWishedYear']):

```

Comment:
where is the else for this?

Reply:
Correct me if I'm wrong but else is not needed I guess. Because only if the bday is today then we send wishes else we don't do anything. So, I don't think and else is a must here.

Reply:
Oh right, got it